src/static/js/bsc_wantanmee.js

```diff
@@ -0,0 +1,117 @@
+
+//TODO
+//const WANTANMEE_BOARDROOM_ABI=
+
+async function main() {  
+  const App = await init_ethers();
+
+  _print(`Initialized ${App.YOUR_ADDRESS}\n`);
+  _print("Reading smart contracts...\n");
+
+ const WANTANMEE_CHEF_ADDR = "0x4c8170476eF5aC641F9a1045adFF75e42E51c33A";
+ const WANTANMEES_CHEF_ADDR = "0xeEBD90C18ADB3a2B288993e6E99F99A6d802e378";
+//TODO:
+// const WANTANMEES_BOARDROOM_ADDR = "0x74eE3b0C391a2190D4fC275cd6C9E9C8dC109240";
+
+ const rewardWantanmeeTokenTicker = "MEE";
+ const rewardWantanmeeShareTokenTicker = "MES";
+
+ const WANTANMEE_CHEF = new ethers.Contract(WANTANMEE_CHEF_ADDR, WANTANMEE_CHEF_ABI, App.provider);
+ const WANTANMEES_CHEF = new ethers.Contract(WANTANMEES_CHEF_ADDR, WANTANMEES_CHEF_ABI, App.provider);
+//TODO: 
+// const WANTANMEE_BOARDROOM = new ethers.Contract(WANTANMEES_BOARDROOM_ADDR, WANTANMEE_BOARDROOM_ABI, App.provider);
+
+ const rewardsWantanmeePerWeek = await WANTANMEE_CHEF.rewardPerBlock() /1e18
+      * 604800 / 3;
+  const rewardsWantanmeeSharePerWeek = await WANTANMEES_CHEF.mesPerBlock() /1e18
+      * 604800 / 3;
+
+//  const currentEpoch = await WANTANMEE_BOARDROOM.epoch();
+  const maxSupplyIncrease = 1000000 // unlimited
+
+//TODO:
+//  const WANTANMEE_BOARDROOM_ORACLE_ADDR = "0x6D1B7249f83f73c71Cd96CD781166a0be15e3CD5";
+//  const rewardTokenAddress = await WANTANMEE_BOARDROOM.dollar();
+//  const lptAddress = "0x284A5D8712C351Ca28417d131003120808dcE48B";
+//  const stakeTicker = "MES";
+//  const rewardTicker = "MEE";
+//  const epochsPerDay = 3;
+//  const decimals = 2;
+//  const ratio = 1.0;
+//  const targetMantissa = 12;
+// 
+  const tokens = {};
+  const prices = await getBscPrices();
+
+  await loadChefContract(App, tokens, prices, WANTANMEE_CHEF, WANTANMEE_CHEF_ADDR, WANTANMEE_CHEF_ABI, rewardWantanmeeTokenTicker,
+    "mee", null, rewardsWantanmeePerWeek, "pendingReward", null, 7);
+  
+  await loadChefContract(App, tokens, prices, WANTANMEES_CHEF, WANTANMEES_CHEF_ADDR, WANTANMEES_CHEF_ABI, rewardWantanmeeShareTokenTicker,
+    "mes", null, rewardsWantanmeeSharePerWeek, "pendingReward", null, 2);
+
+// let br = await loadBoardroom(App, prices, WANTANMEES_BOARDROOM_ADDR, WANTANMEE_BOARDROOM_ORACLE_ADDR, lptAddress, 
+//   rewardTokenAddress, stakeTicker, rewardTicker, epochsPerDay, maxSupplyIncrease, decimals, ratio, targetMantissa)
+// 
+  hideLoading();  
+}
+
+
+
+async function loadChefContract(App, tokens, prices, chef, chefAddress, chefAbi, rewardTokenTicker,
```

Comment:
You should be able to use the existing loadBscChefContract function from bsc_helpers instead of this

Reply:
because our Chef contract doesn't have poolLength() function 
same `SOUP`  finance
so we customize this function a bit